docs/sources/alerting/fundamentals/notifications/alertmanager.md

```diff
@@ -0,0 +1,46 @@
+---
+aliases:
+  - ../../fundamentals/alertmanager/
+  - ../../unified-alerting/fundamentals/alertmanager/
+  - ../../manage-notifications/alertmanager/
```

Comment:
Another request for comments with absolute URLs to help for the future.

Reply:
addressed by https://github.com/grafana/grafana/pull/84248/commits/1133f2d73d48d096fedd2b7f76eec40923e86344